test/integration/webhook/model_test.go

```diff
@@ -147,6 +147,12 @@ var _ = ginkgo.Describe("model default and validation", func() {
 			},
 			failed: true,
 		}),
+		ginkgo.Entry("set filename, allowPatterns and ignorePatterns when modelHub is Huggingface", &testValidatingCase{
```

Comment:
What I mean is adding a model repo, and allowPatters and ignorePatterns both set but the filename is empty, to make sure we will not failed.